framework/resources/Functional/timestamp_functions/timestamp_functions.json

```diff
@@ -0,0 +1,21 @@
+{
+  "testId": "timestampadd_timestampdiff_tests",
+  "type": "group",
+  "description": "Tests for TIMESTAMPADD and TIMESTAMPDIFF functions",
+  "categories": [
+    "functional"
+  ],
+  "matrices": [
+    {
+      "query-file": ".*.sql",
+      "schema": "dfs.drilltestdirtpch01parquet",
+      "output-format": "tsv",
+      "expected-file": ".*.e_tsv",
+      "verification-type": [
+        "in-memory"
+      ]
+    }
+  ],
+  "datasources": [
+  ]
```

Comment:
Hey, I understand you are reusing an existing dataset (TPC-H 0.01). But we recommend each test def file to be self sufficient and independent. So can you look up one of the other test suites which uses this dataset / workspace and provide that in the datasources section?

Reply:
I actually referred to `Functional/filter/selectivity/plan` and `Functional/filter/selectivity/negative/plan` to build the definition file and they didn't seem to have anything mentioned for `datasources`.

Reply:
Done.

Reply:
Yeah, I think we might have missed catching it, when adding that suite. But in general, we would want to be able to run any test suite independently. Thanks for fixing this.